### PR TITLE
docs: cookbook shoutout ready

### PR DESCRIPTION
**Pull request by vanGalilea**
### Summary

Prep. Cookbook App/Docs to be shoutout ready for announcement via social channels

### Test plan

1. Run Cookbook app by following it's updated readme.
2. Ensure readability, cohesion in story and no dead links.

### CODE REVIEW

**Comment by vanGalilea** [comment]
Hey @mdjastrzebski 👋🏻 can you please have a look?
Tnx in advance!

**Review by mdjastrzebski — APPROVED** [review]
🔥🔥🔥

File: examples/cookbook/README.md [hunk 1]
```diff
@@ -1,10 +1,30 @@
-# RNTL Cookbook
+<p align="center">
+  <img alt="banner" src="https://github.com/user-attachments/assets/9f34e38a-32a1-422a-a906-9428036fcd85">
```
Comment (mdjastrzebski) [inline examples/cookbook/README.md:2]: Can you upload this image as a port of the repo?

**Comment by vanGalilea** [comment]
Ready to role @mdjastrzebski, when ready we can do the announcing!

**Review by mdjastrzebski — APPROVED** [review]

**Pull request merged**